week_2/advanced_javascript/question_4/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css" />
+    <title>3 Options layout</title>
+</head>
+<body>
+    <div class="nav-menu_container">
+    <div class="navbar">
+        <div class="option" onclick="toggleMenu(0)">Recent</div>
+        <div class="option" onclick="toggleMenu(1)">Popular</div>
+        <div class="option" onclick="toggleMenu(2)">Trending</div>
```

Comment:
**Suggestion:** The onclick event handlers are directly embedded in the HTML. It's better to separate JavaScript from HTML by using addEventListener in a separate JavaScript file.
```suggestion
        <div class="option" id="recent">Recent</div>
        <div class="option" id="popular">Popular</div>
        <div class="option" id="trending">Trending</div>
```